HW03_Julia.Rmd

```diff
@@ -0,0 +1,105 @@
+---
+title: "HW03_Julia"
+author: "Julia Shangguan"
+date: "7/19/2020"
+output: github_document 
+---
+
+```{r setup, include = TRUE}
+knitr::opts_chunk$set(echo = TRUE, error = TRUE)
+```
+
+## HWO3: Looking at COVID-related data
+
+There is a lot of cool data available from the Pew Research Center. Here is where I downloaded my dataset from: [PewResearchCenter](https://www.journalism.org/dataset/covid-19-late-april-2020/)
+
+**Loading in necessary packages and installing packages:**
+```{r load libraries}
+install.packages('survey') #install this package for analysis of data analysis
+library(survey)
+install.packages('tidyverse')
+library(tidyverse)
+library("ggplot2")
+library(foreign) #needed to import my dataset
+library(scales) #needed to later convert my x-axis labels from decimal to percent
+```
+
+**Importing my dataset as an R data.frame:**
+```{r}
+Covid_data <- read.spss("Pathways April 2020 (ATP W66).sav", to.data.frame = TRUE) 
+```
+
+**The survey question I will be examining is**: "All in all, do you think that the coronavirus outbreak has been…"
+
+* made a bigger deal than it really is
+
+* made a smaller deal than it really is
+
+* approached about right
+
+* no answer
+
+
+A lot of my time was spent trying to understand how this dataset was organized. I found this [document](https://www.pewresearch.org/wp-content/uploads/2018/05/Codebook-and-instructions-for-working-with-ATP-data.pdf) that explained what the demographic profile variables meant. The variables corresponding to each survey question response were included in this document titled [Pathways April 2020 ATP W66](Pathways_April 2020_ATPW66_ Topline.pdf) which comes with the dataset download. I am interested in responses as stratified by age (F_AGECAT) and self-reported political party (F_PARTY_FINAL).
+
+```{r}
+#names(Covid_data) #variables corresponding to each survey question
+```
+
+I will be plotting weighted survey estimates based on the variable WEIGHT_W66 because "data for all Pew Research Center reports are analyzed using this weight."
+
+First, I am cleaning up my dataframe and limiting the dataset to only the columns I need. 
+```{r}
+#limit to select columns, also renaming some columns
+select_Covid_data <- Covid_data %>%
+  select(weight = WEIGHT_W66, response = COVIDDEAL_W66, Age = F_AGECAT, Party = F_PARTY_FINAL) 
+
+head(select_Covid_data) #checking that it worked
+```
+
+I am rearranging the data so that I can easily calculate weighted summary stats by each demographic group by reshaping data to a "long" format. I'm not sure how to describe this, but [this article](https://medium.com/pew-research-center-decoded/using-tidyverse-tools-with-pew-research-center-survey-data-in-r-bdfe61de0909) touches upon it; this article was also just super helpful in general for formatting Pew Research data sets. 
```

Comment:
Thank you for this link! It really helped with understanding how to code responses from surveys.